README.md

```diff
@@ -0,0 +1,74 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] You have done well providing the link to the front end on the PR summary note. However, visitors to your project will not be able to access the link to your frontend if that is the only where is it located. Kindly add the link to this readme as well for proper documentation.

- [ ] In your frontend repo kindly include the link to the backend repo in the readme also for better documentation.